### ISSUE TITLE
pylint 2.12.0 produces traceback with pylint-fixme-info

### ISSUE THREAD

**Issue by sergiusens** [PyCQA/pylint#5390]
### Bug description

Seems like the latest pylint is not working well with pylint-fixme-info and produces a traceback

### Command used

```shell
pylint craft_providers
```

### Pylint output

```shell
Traceback (most recent call last):
  File "/home/sergiusens/repo/canonical/craft-providers/.direnv/python-3.9.7/bin/pylint", line 8, in <module>
    sys.exit(run_pylint())
  File "/home/sergiusens/repo/canonical/craft-providers/.direnv/python-3.9.7/lib/python3.9/site-packages/pylint/__init__.py", line 24, in run_pylint
    PylintRun(sys.argv[1:])
  File "/home/sergiusens/repo/canonical/craft-providers/.direnv/python-3.9.7/lib/python3.9/site-packages/pylint/lint/run.py", line 345, in __init__
    linter.load_plugin_modules(plugins)
  File "/home/sergiusens/repo/canonical/craft-providers/.direnv/python-3.9.7/lib/python3.9/site-packages/pylint/lint/pylinter.py", line 595, in load_plugin_modules
    module = astroid.modutils.load_module_from_name(modname)
  File "/home/sergiusens/repo/canonical/craft-providers/.direnv/python-3.9.7/lib/python3.9/site-packages/astroid/modutils.py", line 218, in load_module_from_name
    return importlib.import_module(dotted_name)
  File "/usr/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/sergiusens/repo/canonical/craft-providers/.direnv/python-3.9.7/lib/python3.9/site-packages/pylint_fixme_info/__init__.py", line 8, in <module>
    from pylint.message import MessagesHandlerMixIn
ImportError: cannot import name 'MessagesHandlerMixIn' from 'pylint.message' (/home/sergiusens/repo/canonical/craft-providers/.direnv/python-3.9.7/lib/python3.9/site-packages/pylint/message/__init__.py)
make: *** [Makefile:97: test-pylint] Error 1
```

### Expected behavior

Same output as 2.11

```
************* Module craft_providers.bases.instance_config
craft_providers/bases/instance_config.py:65:9: I1511: TODO: Replace test / cat usage with an improved (fixme-info)
************* Module craft_providers.lxd.launcher
craft_providers/lxd/launcher.py:168:9: I1511: TODO: warn (or auto clean) if ephemeral or map_user_uid is mismatched. (fixme-info)
************* Module craft_providers.lxd.lxd_instance
craft_providers/lxd/lxd_instance.py:324:22: R1735: Consider using {} instead of dict() (use-dict-literal)
craft_providers/lxd/lxd_instance.py:386:37: R1735: Consider using {} instead of dict() (use-dict-literal)
craft_providers/lxd/lxd_instance.py:387:53: R1735: Consider using {} instead of dict() (use-dict-literal)
************* Module craft_providers.util.os_release
craft_providers/util/os_release.py:50:31: R1735: Consider using {} instead of dict() (use-dict-literal)
************* Module craft_providers.multipass._launch
craft_providers/multipass/_launch.py:60:9: I1511: TODO: Warn if existing instance doesn't match cpu/disk/mem specs. (fixme-info)
************* Module craft_providers.multipass.multipass
craft_providers/multipass/multipass.py:202:56: R1734: Consider using [] instead of list() (use-list-literal)
************* Module craft_providers.multipass.multipass_instance
craft_providers/multipass/multipass_instance.py:244:36: R1735: Consider using {} instead of dict() (use-dict-literal)
************* Module craft_providers.multipass.installer
craft_providers/multipass/installer.py:54:5: I1511: TODO: Multipass needs time after being installed or errors could happen on (fixme-info)

------------------------------------------------------------------
Your code has been rated at 9.95/10 (previous run: 9.95/10, +0.00)
```

### Pylint version

```shell
running pylint version produces the same traceback

astroid==2.9.0
...
pylint==2.12.0
pylint-fixme-info==1.0.2
pylint-pytest==1.1.2


Python 3.9.7
```

### OS / Environment

Ubuntu 20.04 and 21.10

### Additional dependencies

alabaster==0.7.12
astroid==2.9.0
attrs==21.2.0
autoflake==1.4
Babel==2.9.1
backports.entry-points-selectable==1.1.1
black==21.11b1
bleach==4.1.0
certifi==2021.10.8
cffi==1.15.0
charset-normalizer==2.0.8
click==8.0.3
codespell==2.1.0
colorama==0.4.4
coverage==6.1.2
-e git+git@github.com:canonical/craft-providers.git@ad448804c21efdb5891072c601071066ad17d2a0#egg=craft_providers
cryptography==36.0.0
distlib==0.3.3
docutils==0.17.1
filelock==3.4.0
flake8==4.0.1
idna==3.3
imagesize==1.3.0
importlib-metadata==4.8.2
iniconfig==1.1.1
isort==5.10.1
jeepney==0.7.1
Jinja2==3.0.3
jsonpointer==2.2
keyring==23.3.0
lazy-object-proxy==1.6.0
MarkupSafe==2.0.1
mccabe==0.6.1
mypy==0.910
mypy-extensions==0.4.3
packaging==21.3
pathspec==0.9.0
pkginfo==1.8.1
platformdirs==2.4.0
pluggy==1.0.0
py==1.11.0
pycodestyle==2.8.0
pycparser==2.21
pydantic==1.8.2
pydocstyle==6.1.1
pyflakes==2.4.0
Pygments==2.10.0
pylint==2.12.0
pylint-fixme-info==1.0.2
pylint-pytest==1.1.2
pyparsing==3.0.6
pytest==6.2.5
pytest-mock==3.6.1
pytest-subprocess==1.3.2
pytz==2021.3
PyYAML==6.0
readme-renderer==30.0
regex==2021.11.10
requests==2.26.0
requests-toolbelt==0.9.1
requests-unixsocket==0.2.0
rfc3986==1.5.0
SecretStorage==3.3.1
six==1.16.0
snowballstemmer==2.2.0
Sphinx==4.2.0
sphinx-autodoc-typehints==1.12.0
sphinx-jsonschema==1.16.11
sphinx-pydantic==0.1.1
sphinx-rtd-theme==1.0.0
sphinxcontrib-applehelp==1.0.2
sphinxcontrib-devhelp==1.0.2
sphinxcontrib-htmlhelp==2.0.0
sphinxcontrib-jsmath==1.0.1
sphinxcontrib-qthelp==1.0.3
sphinxcontrib-serializinghtml==1.1.5
toml==0.10.2
tomli==1.2.2
tox==3.24.4
tqdm==4.62.3
twine==3.6.0
types-PyYAML==6.0.1
types-requests==2.26.0
types-setuptools==57.4.3
typing-extensions==4.0.0
urllib3==1.26.7
virtualenv==20.10.0
webencodings==0.5.1
wrapt==1.13.3
zipp==3.6.0

**Comment by Pierre-Sassoulas** [comment]
Thank you for opening the issue. There's no easy fix as the ``MessageHandlerMixin`` was removed in 2.12 (we considered that a private API). It can probably be replaced by ``Pylinter`` (?)

Also it seems  ``MessageHandlerMixin`` [is imported but not actually used in the code](https://github.com/mxmeinhold/pylint-fixme-info/blob/main/pylint_fixme_info/__init__.py#L8). A fix would be to remove the import unless I miss something obvious.

**Comment by DanielNoord** [comment]
@sergiusens Would you able to report this against https://github.com/mxmeinhold/pylint-fixme-info/? We can't actually do much with this issue, although we can keep it open for visibility until the real issue has been resolved!

**Comment by Pierre-Sassoulas** [comment]
Closing as won't fix, it's a private API, and there's an easy fix (removing the import as it's not used).

**Comment by mr-cal** [comment]
The downstream regression was resolved in [pylint-fixme-info-v1.0.3](https://github.com/mxmeinhold/pylint-fixme-info/releases/tag/v1.0.3)

**Comment by Marcws10** [comment]
bom dia, alguém sabe como resolver isso. quando tento abrir o wapiti aparece esse erro

└─$ wapiti
Traceback (most recent call last):
  File "/usr/bin/wapiti", line 33, in <module>
    sys.exit(load_entry_point('wapiti3==3.0.4', 'console_scripts', 'wapiti')())
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/bin/wapiti", line 25, in importlib_load_entry_point
    return next(matches).load()
           ^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/importlib/metadata/__init__.py", line 202, in load
    module = import_module(match.group('module'))
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1206, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1178, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1149, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/usr/lib/python3/dist-packages/wapitiCore/main/wapiti.py", line 41, in <module>
    from wapitiCore.language.language import _
  File "/usr/lib/python3/dist-packages/wapitiCore/language/language.py", line 62, in <module>
    lan = gettext.translation(
          ^^^^^^^^^^^^^^^^^^^^
TypeError: translation() got an unexpected keyword argument 'codeset'
